### DISCUSSION TITLE
`formulae.brew.sh` search index outdated

### DISCUSSION THREAD

**Discussion by nihaals** [Homebrew/discussions#4000] [Everyday usage]
It seems like the Algolia search index hasn't been updated in a while as new formulae (e.g. `python@3.11`) and casks (e.g. `anytype`) aren't showing up in search (but the pages exist). Is this process meant to run automatically every so often or is this manual?

**Comment by Bo98** [comment]
We're aware and it required contacting them about it so progress has been slow unfortunately. It should be dealt with in the next day or two hopefully.

**Reply by nihaals** [reply]
Has there been an update on this?

**Reply by SMillerDev** [reply]
I just checked and the crawl succeeded fine recently

**Reply by nihaals** [reply]
I've found a few that don't show in search results for me:
- [`python@3.11`](https://formulae.brew.sh/formula/python@3.11)
- [`superlist`](https://formulae.brew.sh/cask/superlist) (cask)
- [`anytype`](https://formulae.brew.sh/cask/anytype) (cask)

For example:
![image](https://user-images.githubusercontent.com/18350092/215526634-1404bc31-45db-4f52-ab5e-c785daf532ac.png)

![image](https://user-images.githubusercontent.com/18350092/215526261-60467071-9894-49b5-aaf6-840a72c9579a.png)

**Reply by SMillerDev** [reply]
They're definitely in the data that the search is based on. This seems to go missing in the Algolia magic.

**Reply by nihaals** [reply]
Something's definitely wrong and it does appear to be the same list of casks and formulae that weren't searchable before as if the index hasn't *actually* updated

---

[Cask](https://formulae.brew.sh/cask/capslocknodelay)

![image](https://user-images.githubusercontent.com/18350092/215836481-e79a6b02-3459-44b1-a2f9-ddf0a614af17.png)

**Reply by SMillerDev** [reply]
It seems I've missed some information from Algolia. "Here's your new index" actually meant "here's your new index but it's not hooked up yet". This should be fixed after https://github.com/Homebrew/brew.sh/pull/927
